### ISSUE TITLE
Types of hardware modules for use with LoRa & Xbee

### ISSUE THREAD

**Issue by olenamolla** [ngcp-project/ngcp-project.github.io#25]
1. LoRa Modules: HopeRF RFM95/96/97/98, Semtech SX1276/77/78/79, RAK Wireless RAK811(?)
2. Xbee Modules: Xbee Pro 900HP, Xbee 3 Pro, Xbee SX

Xbee modules are more expensive. It also depends which microcontroller we will be using; we might need to purchase shields and antennas separately.

**Comment by marked01one** [comment]
Xbee RR Pro is also really good module if you wanna check that out. Advertised range is up to 2 miles [https://www.sparkfun.com/products/22631](https://www.sparkfun.com/products/22631)

**Comment by marked01one** [comment]
I noticed that some of these are like microchips designed for LoRa, but make sure to focus on actual LoRa modules that can be connected to something like an Arduino or Raspberry Pi or to normal USB via some adapter.

**Comment by olenamolla** [comment]
Yes, you are right. We either can get the complete modules or purchase all parts of the module separately. But ExpressLRS protocol requires specific hardware. 

Here is the examples of a complete LoRa module:
[LoRa SX1278](https://www.amazon.com/Comimark-SX1278-Wireless-Mental-Arduino/dp/B07W6ZPH7D/ref=sr_1_1?crid=OXJM2P0OCCRC&dib=eyJ2IjoiMSJ9.NpIPDXlT8IF3ZZj6M_MwZHPRypfNTO2ySCaGce1ZI3UK-7JCKY2bG9BY3ovzRr3NtfH9bxdVVaBhDFwJAFlJ65-XuMjkeqm8b3bYoQ7hqtibxyBaqAmcx0pVvg4u086Lk7KlixFcEkGLF4ZOYCQi3THq8FxXB-W5ElCubQxGDI8WD7V7xclsO-FDVSWJB7uSjCzWkbaXftyeOt7yG_Ye63lTChzLGkscT7vV6rsZd7OHFkdwYAgvuf1SnzKPB_12zcZJKhGBDgi8nFlFNxvhWEWsPC5NtG8zSfjPnOB7nGE.dbl0iDH8f9tzx4dIfVuSwIRPOcV75HdrB975qQEqku0&dib_tag=se&keywords=complete+LoRa+module+for+arduino&qid=1718996978&s=electronics&sprefix=complete+lora+module+for+arduino%2Celectronics%2C139&sr=1-1)

**Comment by marked01one** [comment]
I found this LoRa module that is an upgrade of another module I saw a lot on YouTube: [Reyax RYLR993-Lite](https://www.digikey.com/en/products/detail/reyax/RYLR993-Lite/22237834) 

It's an upgrade to their [RYLR998](https://www.amazon.com/REYAX-RYLR998-Interface-Antenna-Transceiver/dp/B099RM1XMG)

Here's some videos with people testing them out:
https://www.youtube.com/watch?v=IEmlHATj-Qc
https://www.youtube.com/watch?v=d9u6QIyR8a8
https://www.youtube.com/watch?v=9azEfCQNhSA
